src/write_gcm.c

```diff
@@ -4162,6 +4169,11 @@ static int wg_write(const data_set_t *ds, const value_list_t *vl,
       queue->tail = NULL;
     }
     --queue->size;
+    ++queue->drop_count;
+    if ((queue->drop_count % QUEUE_DROP_REPORT_LIMIT) == 0) {
+      WARNING("Queue dropped %d metric points due to dispatch backlog.",
```

Comment:
Let's also include the `queue_name` in the message so we know which queue is dropping metrics...